packages/dna_tools.py

```diff
@@ -0,0 +1,150 @@
+        -rna seqeunce in str format.
+    """
+    rna = ''
+    for nucl in seq:
+        rna += dna2rna[nucl]
+    return rna
+    
+def reverse(seq: str) -> str:
+    """
+    Transforms dna/rna sequence into it's reverse.
+    Arguments: 
+        -seq (str): dna/rna seqeunce.
+    Return: 
+        -str - reverse dna/rna sequence.
+    """
+    return seq[::-1]
+
+def complement(seq: str) ->str:
+    """
+    Transforms dna/rna sequence into it's complement.
+    Arguments: 
+        -seq (str): dna/rna seqeunce.
+    Return: 
+        -str - complement dna/rna sequence.
+    """
+    cstrand = ''
+    if 'u' in seq.lower():
+        for nucl in seq:
+            cstrand += rna2crna[nucl]
+    else:
+        for nucl in seq:
+            cstrand += dna2cdna[nucl]
+    return cstrand
+
+def reverse_complement(seq: str) -> str:
+    """
+    Transforms dna/rna sequence into it's reverse complement.
+    Arguments: 
+        -seq (str): dna/rna seqeunce.
+    Return: 
+        -str - reverse complement dna/rna sequence.
+    """
+    reverse_cstrand = complement(reverse(seq))
+    return reverse_cstrand
+        
+
+def count_gc(seq: str) -> float:
+    """
+    Counts gc content of sequence.
+    Arguments: 
+        -seq (str): dna/rna seqeunce.
+    Return: 
+        -float - percent of GC content in sequence.
+    """
+    gc = seq.lower().count('g') + seq.lower().count('c')
+    answer = gc / len(seq) * 100
+    return round(answer, 2)
+
+def dna_or_rna(seq: str) -> str:
+    """
+    Determines whether the input is DNA or RNA.
+    Arguments:
+        -seq (str): dna/rna seqeunce.
+    Return:
+        -str - dna or rna based on content.
+    """
+    if set(seq.lower()) == set('atgc'):
+        return 'dna'
+    elif set(seq.lower()) == set('augc'):
+        return 'rna'    
+
+def get_sequence_length(seq: str) -> int:
+    """
+    Counts the length of the sequence
+    Arguments:
+        -seq (str): dna/rna seqeunce.
+    Return:
+        -int - length of the sequence.
+    """
+    return len(seq)
+
+commands = {'transcribe': transcribe, 'reverse': reverse, 'complement':complement, 'reverse_complement': reverse_complement, 'count_gc': count_gc, 
+'dna_or_rna': dna_or_rna, 'get_sequence_length': get_sequence_length}
+
+def check(args: list[str]):
+    """
+    Checks list of sequences for correct format. 
+    Arguments:
+        -args (list) - Input format: at least 1 sequence and command, command on last position of list,
+        only DNA/RNA seqeunces.
+    Return:
+        -ValueError - if something does not fit arguments input format.
+    """
+    if len(args)  < 2:
+        raise ValueError('Input sequences and command')
+    elif args[-1] not in commands:
+        raise ValueError('No such command')
+    else:
+        for seq in args[:-1]:
+            if set(seq.lower()) == set('autgc') and set(seq.lower()) != set('atgc') and set(seq.lower()) != set('augc'):
+                raise ValueError('Input dna or rna')
```

Comment:
```suggestion
                raise ValueError('Input dna or rna')
                
```

функции принято разделять пустями строками

но в основном тут ставлю комментарий, тк увидела, что в дз2 Антон сюда внес комментарий относительно сообщения ошибки, исправления нет

(кстати, про нейминг констант он тоже писал, и это не исправлено)